src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -3417,7 +3414,7 @@ private bool CheckDuplicateFilePaths(DiagnosticBag diagnostics)
         }
 
         /// <returns><see langword="true"/> if duplicate interceptors are present in the compilation. Otherwise, <see langword="false" />.</returns>
-        private bool CheckDuplicateInterceptions(DiagnosticBag diagnostics)
+        internal bool CheckDuplicateInterceptions(BindingDiagnosticBag diagnostics)
```

Comment:
nit: the comment above should probably say "interceptions" instead of "interceptors"